### PR TITLE
Simplify cache clean

### PR DESCRIPTION
**Pull request by giorgiosironi**
Avoids rsync, since it's a small performance improvement over simple commands but it's not going to solve our performance problems on cache cleaning.

### CODE REVIEW

**Comment by lsh-0** [comment]
is this still relevant? if so, I'm happy to resolve conflicts and merge it in while IIIF changes are still fresh  in my head. Otherwise, we should close it.

**Comment by giorgiosironi** [comment]
Yes, I can solve the conflicts. Essentially `rsync` was an attempt to improve performance of deleting zillions of files, but it doesn't really do a difference (what was I thinking?)

**Comment by lsh-0** [comment]
> but it doesn't really do a difference (what was I thinking?)

I was wondering that too ... perhaps moving the cache directory out of the way and running a `rm -f` with a low priority `nice` value would be another approach.

rsync still feels like magic to me, I was curious what it could do to perform better

File: salt/iiif/config/usr-local-bin-loris-cache-clean-hard [hunk 1]
```diff
@@ -5,9 +5,6 @@
 
 set -e
 
-# rsync... http://www.slashroot.in/which-is-the-fastest-method-to-delete-files-in-linux
-# https://web.archive.org/web/20130929001850/http://linuxnote.net/jianingy/en/linux/a-fast-way-to-remove-huge-number-of-files.html
-BLANK_DIR="{{ pillar.iiif.loris.storage}}/blank"
```
Comment (lsh-0) [inline salt/iiif/config/usr-local-bin-loris-cache-clean-hard]: I'll remove references to the `blank` dir in the cleanup PR

**Pull request merged**